### PR TITLE
Undervolt: Make timer optional

### PR DESCRIPTION
**Pull request by Atemu**
###### Motivation for this change

The undervolt timer applies the undervolt every 30s which spams the journal. IIRC this was done because the undervolt would not survive certain events (reboot, resume).

I have been running these patches for ~3 months and the undervolt holds perfectly on my Broadwell convertible. I don't know if he tested them long-term but they also worked on my friend's Coffelake laptop.  
More test results welcome.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Lassulus** [comment]
hmm, this could break the setup for some people. Maybe add a notice to https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2009.xml ?

**Comment by Atemu** [comment]
Sure, I've never written docbook before but I'm sure I can manage.

How can I build the release notes to verify they look correct?

**Comment by Lassulus** [comment]
Oh sorry, I missed the notification. 
In the root of nixpkgs you can run `nix-build nixos/release.nix -A manual.x86_64-linux` this outputs a path to the nix-store. in the path you will find the html under `/share/doc/nixos/index.html`

**Comment by Atemu** [comment]
Bump

File: nixos/doc/manual/release-notes/rl-2009.xml [hunk 1]
```diff
@@ -666,6 +666,13 @@ systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/var/www" ];
      since Nextcloud doesn't support upgrades across multiple major versions.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The <literal>undervolt</literal> option longer needs to apply its settings
```
Comment (Lassulus) [inline nixos/doc/manual/release-notes/rl-2009.xml:671]:
```suggestion
      The <literal>undervolt</literal> option no longer needs to apply its settings
```

**Pull request merged**